### ISSUE TITLE
I wrapped the application and it worked, but it doesn't seem to work

### ISSUE THREAD

**Issue by songmw90** [msintuneappsdk/intune-app-wrapping-tool-ios#80]
We have already wrapped our packaged app and we have successfully wrapped it.
However, I heard a report from the client that MAM was not running properly, and I found that the wrapped app was different when the screen was turned on.

![screen](https://user-images.githubusercontent.com/4706240/169436200-fd5f8de1-53c2-48fb-bc22-18184aa11d12.png)

When I checked, a similar screen appeared for the Android-wrapped app. However, in the iOS app, this screen does not appear, only the version information is changed.

What more can I check? It doesn't seem to be controlled by the screen as well as the MAM policy. Unable to open document received from Outlook

**Comment by vigavadi** [comment]
Hey @songmw90 This screen wouldn't appear every time. If you are able to access you app, then it means the policies should have successfully applied. What issues are you seeing with the document?

**Comment by songmw90** [comment]
If you directly install the Android wrapped app and run the app, it will appear like the screenshot above.

However, in the case of iOS, even when directly installed, it works the same as an unwrapped app. I think Android's execution plan seems to be right.

**Comment by vigavadi** [comment]
@songmw90 Could you send me logs as per this document: [Logs for Wrapped Apps](https://docs.microsoft.com/en-us/mem/intune/developer/app-wrapper-prepare-ios#collecting-logs-for-your-wrapped-applications-from-the-device) to vigavadi@microsoft.com?

**Comment by songmw90** [comment]
The diagnostic console menu is present, but doesn't open when starting the app.
As a special note, intune wrapping fails in versions 14.x and 16.0. We tested it with a successful app on version 15.1.4

**Comment by vigavadi** [comment]
Hey @songmw90, does the diagnostic console not open even after turning the toggle on from the settings and then restarting the app?

![IMG_0001](https://user-images.githubusercontent.com/62442249/173437836-87e047e6-1240-4aea-acce-f71b2e48b9a0.jpg)

**Comment by songmw90** [comment]
Yes it is

**Comment by vigavadi** [comment]
@songmw90 Could you try out version 16.0.2 and see if that fixes your issue?

**Comment by songmw90** [comment]
We had success with the wrap tool version 16.0.2. (Before that, lapping failed at all)
But still the first screen is the same, even with all options enabled, the console doesn't open

Is the above request still valid?

**Comment by songmw90** [comment]
No, it was not resolved. it's late  night here. I will prepare the report tomorrow and send it to you.

**Comment by vigavadi** [comment]
@songmw90 Thanks for the info, this is really helpful for debugging issues. One follow-up question, does you app support scenes or does your application still rely on the AppDelegate to manage the single window object? If your app is not configuring scenes, could you remove this from your Info.plist file and try again?
<img width="792" alt="image" src="https://user-images.githubusercontent.com/62442249/174352628-8e5731da-2e75-4cf1-a9ac-637f5d2f8cc5.png">

**Comment by songmw90** [comment]
We are currently on Saturday. The code is at the company, so I'll check it out after the weekend and let you know. However, our app has been maintained since iOS5. So, as far as I know, it doesn't use scenes, it uses AppDelegate .

**Comment by songmw90** [comment]
The Application Scene Manifest (Generation) value of our app is No for both debug / release.
As you said, when I deleted the column and wrapped it, the first screen changed and the console appeared properly.

However, we can't delete this option right away because we don't know what effect it will have on our app. Is this issue going to be supported by the wrapping tool or should we delete the column?

**Comment by vigavadi** [comment]
@songmw90 Great to know that it worked. We'll work on a fix for this. But in general, if your app doesn't use scenes, then removing UIApplicationSceneManifest shouldn't be any problem. The UIApplicationSceneManifest is only useful if you have the UISceneConfigurations dictionary set. You can refer to the Apple Documentation for [UIScenes](https://developer.apple.com/documentation/uikit/app_and_environment/scenes/specifying_the_scenes_your_app_supports?language=objc)

**Comment by songmw90** [comment]
The app works, but crashes. There was no crash in the old version.

I sent the log and video by e-mail, so please check

**Comment by vigavadi** [comment]
@songmw90 This has been fixed in 16.1.1
